### DISCUSSION TITLE
Getting error after running ungoogled-chromium in Manjaro

### DISCUSSION THREAD

**Discussion by miguelcarcamov** [Eloston/ungoogled-chromium#1640] [Q&A] [answered]
Hello, when I run chromium and then I start to using it (like opening a tab or any action) I get this message from the terminal: 

```
[1506272:1506286:0914/131954.381276:ERROR:ev_root_ca_metadata.cc(841)] Failed to register OID: 0
Segmentation fault (core dumped)
```
I have tried both installing from the AUR and installing the binary and in both cases I get the same result.

**Comment by networkException** [comment, marked as answer]
This has been moved to https://github.com/ungoogled-software/ungoogled-chromium-archlinux/issues/145
